src/bundle/Resources/public/scss/_page-title.scss

```diff
@@ -1,15 +1,35 @@
 .ez-page-title {
     padding-bottom: 1rem;
+    display: grid;
+    grid-template-columns: 1fr 40px;
+    grid-column-gap: 10px;
+    grid-template-rows: auto;
+    grid-template-areas:
+        "content-item bookmark"
+        "content-type-name .";
 
     h1 {
```

Comment:
Can you reuse CSS classes you added to the Twig file? It will allow to avoid styling by tag selectors.